packages/@datapull/http-api-origin/index.js

```diff
@@ -48,6 +123,14 @@ class HttpApiOrigin {
         options.auth = `${config.username}:${config.password}`;
       }
 
+      if (this.config.headers) {
+        options.headers = {};
+        Object.keys(this.config.headers).forEach(k => {
```

Comment:
My personal preference is that we use `for .. of` for loops instead.

Reply:
fixed